frontend/public/components/utils/category-filter-utils.ts

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable no-undef */
+import * as _ from 'lodash-es';
+
+export class CategoryFilterUtils {
+  static CATEGORY_URL_PARAM = 'category';
+  static KEYWORD_URL_PARAM = 'keyword';
+  static FILTER_SEPARATOR = ',|,';
```

Comment:
I'd rather serialize a JSON array than trying to use our own format

Reply:
Fixed